### PR TITLE
Issue 24 rest api

### PR DESCRIPTION
**Pull request by dbyrns**

### CODE REVIEW

**Comment by tomLandry** [comment]
Wow, nice merge in perspective.

File: environment.yml [hunk 1]
```diff
@@ -38,8 +39,16 @@ dependencies:
...
 - pip:
   - genshi==0.7
   - pyramid-rpc
+  - pyramid-celery
   - sphinx-autoapi
   - -e git+https://github.com/huard/sphinx-autodoc-pywps.git#egg=sphinx_autodoc_pywps
+  - https://github.com/geopython/OWSLib/archive/master.zip
+  - https://github.com/geopython/pywps/archive/master.zip
+  - cornice_swagger
+  - cornice
```
Comment (fmigneault) [inline environment.yml]: duplicate `cornice`

File: environment.yml [hunk 2]
```diff
@@ -38,8 +39,16 @@ dependencies:
...
+- cornice
+- colander
 - pip:
   - genshi==0.7
   - pyramid-rpc
+  - pyramid-celery
   - sphinx-autoapi
   - -e git+https://github.com/huard/sphinx-autodoc-pywps.git#egg=sphinx_autodoc_pywps
+  - https://github.com/geopython/OWSLib/archive/master.zip
+  - https://github.com/geopython/pywps/archive/master.zip
```
Comment (fmigneault) [inline environment.yml]: still points to `master`
uncomment above `pywps=4.0.0` and remove here

**Review by fmigneault — COMMENTED** [review]
requirements have some possible issues as previously discussed

**Comment by dbyrns** [comment]
Relevant content has been moved and greatly improved in https://github.com/crim-ca/weaver.

**Pull request closed without merging**